calabash-cucumber/lib/calabash-cucumber/rotation_helpers.rb

```diff
@@ -32,141 +26,26 @@ def rotation_candidates
       # @note Refer to Apple's documentation for clarification about left vs.
       #  right landscape orientations.
       #
-      # @note For legacy support the `dir` argument can be a String or Symbol.
-      #  Please update your code to pass a Symbol.
-      #
-      # @note For legacy support `:top` and `top` are synonyms for `:up`.
-      #  Please update your code to pass `:up`.
-      #
-      # @note For legacy support `:bottom` and `bottom` are synonyms for `:down`.
-      #  Please update your code to pass `:down`.
-      #
-      # @note This method generates verbose messages when full console logging
-      #  is enabled.  See {Calabash::Cucumber::Logging#full_console_logging?}.
-      #
       # @param [Symbol] dir The position of the home button after the rotation.
       #  Can be one of `{:down | :left | :right | :up }`.
-      #
-      # @return [Symbol] The orientation of the button when all rotations have
-      #  been completed.  If there is problem rotating, this method will return
-      #  `:down` regardless of the actual home button position.
-      #
-      # @todo When running under UIAutomation, we should use that API to rotate
-      #  instead of relying on playbacks.
       def rotate_home_button_to(dir)
-        dir_sym = dir.to_sym
```

Comment:
@krukow 

Are the changes here too broad?  It looks like if the user is _not_ running with UIA, rotations will not work.